SR2025/2024-11-27-nov-update.md

```diff
@@ -0,0 +1,73 @@
+---
+to: SR2025 teams
+subject: SR2025 November Update
+---
+
+Hi,
+
+With the competition cycle in full swing we hope your team has found its feet.
+
+We have a few updates and reminders for you:
+
+* We have released an update to the simulator, fixing a few bugs
+* The first [Challenge submission deadline][challenge-deadline] is in a week and a half
+* We're running some virtual friendlies in a couple of weeks
+* The next [Tech Day][tech-days] is in Cambridge, [in January][cambridge-tech-day-january] -- please [sign up][tech-day-signup] if you're interested
+
+## Simulator Update
+
+Using competitor feedback we have released a new version of the simulator with some fixes. This can be downloaded from [our GitHub](https://github.com/srobo/sbot_simulator/releases/download/2025.1.1/sbot-simulator-2025.1.1.zip) or using the link in [the simulator docs](https://studentrobotics.org/docs/simulator/setting_up_simulator).
+
+We've fixed a few issues around the run script not being able to find Webots and ensured that you have all the libraries available to you that are on your robots. These libraries are listed [on our docs](https://studentrobotics.org/docs/kit/brain_board/python_libraries).
+
+Please note that the simulator requires a Python version between 3.9 and 3.12. This is due to an issue with installing the libraries on Python 3.13. We recommend using Python 3.11, the same as is installed on your robot. Downloads for all operating systems are available from [the Python website](https://www.python.org/downloads/release/python-3119/).
```

Comment:
Can we link to the specific downloads for MacOS and Windows as we have in the Discord post.

Reply:
This section (and whole email) is already quite long. Given that it won't be the supervisors who are doing the downloading and that we e.g: don't know the target bitness/OS version I think simpler is better here. Plus letting people learn how to navigate a file list to work out the right one is probably a good thing.